spec/ace.md

```diff
@@ -383,6 +383,33 @@ Small quantities can be represented directly as decimals (e.g., 0.3), or using m
 
 **NOTE**: Network limits MUST NOT apply to localhost communication between apps in a pod.
 
+### Environment Isolators
+
+An application or a group of applications may often require some tweaking to the environment where they run, in order to perform in an optimal way.
```

Comment:
Awkward wording I think. I'll suggest "An application may want or need some changes to the system environment it is run in."

At the least, nix often.